Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -63,6 +63,15 @@ installation of the ``kube-proxy`` add-on:
       kubectl -n kube-system delete ds kube-proxy
       iptables-restore <(iptables-save | grep -v KUBE)
 
+For existing installations with ``kube-proxy`` running as a DaemonSet, remove it
+by using the following commands:
+
+.. code:: bash
+
+      kubectl -n kube-system delete ds kube-proxy
+      # Run on each node:
+      iptables-restore <(iptables-save | grep -v KUBE)
+
```

Comment:
nit: it's not clear to me when we use shell-session and when we use bash, but maybe `. code-block:: shell-session` works better here?

Reply:
:shrug:
```
$~/sandbox/gopath/src/github.com/cilium/cilium/Documentation [02:19:33]
> gg '.. code:: bash' | wc -l
183
$~/sandbox/gopath/src/github.com/cilium/cilium/Documentation [02:19:51]
> gg '.. code:: shell-session' | wc -l
1
```
Do you know what's the difference?

Reply:
First,  it's `.code-block:: shell-session` rather than `.code:: shell-session` (see #13704):
```
$ git grep '.. code-block:: shell-session' | wc -l
$ 76
```

AFAIU, `code-block:: shell-session` marks (and highlights) interactive shell sessions, which `code:: bash` is meant for bash scripts.

Reply:
Aha, then we should change everywhere. For this PR, I'd prefer to stay consistent with the rest of the doc page.

Reply:
Yes, if there is output from the commands then shell-session makes sense because the command output does not follow bash syntax.